core/src/main/resources/hudson/model/Api/index.jelly

```diff
@@ -98,58 +98,47 @@ THE SOFTWARE.
         -->
       </dl>
 
-      <p>
-        For more information about remote API in Jenkins, see
-        <a href="https://jenkins.io/redirect/remote-api">the documentation</a>.
-      </p>
+      <p>For more information about remote API in Jenkins, see <a href="https://jenkins.io/redirect/remote-api">the documentation</a>.</p>
```

Comment:
@daniel-beck Again. Is this blocking the PR merge? If so, I'll revert the changes.